src/mscorlib/shared/System/ReadOnlySpan.cs

```diff
@@ -280,7 +280,7 @@ public override int GetHashCode()
         /// <summary>
         /// Returns a <see cref="String"/> with the name of the type and the number of elements
         /// </summary>
-        public override string ToString() => "System.Span[" + Length.ToString() + "]";
+        public override string ToString() => string.Format("System.ReadOnlySpan<{0}>[{1}]", typeof(T).Name, Length);
```

Comment:
Nit: We are using the field everywhere inside the implementation. Why not to use here as well?

Reply:
Sure.